configure.ac

```diff
@@ -1485,7 +1485,7 @@ if test "$use_natpmp" != "no"; then
   CPPFLAGS="$TEMP_CPPFLAGS"
 fi
 
-if test "$build_bitcoin_wallet$build_bitcoin_cli$build_bitcoin_tx$build_bitcoind$bitcoin_enable_qt$use_tests$use_bench$enable_fuzz_binary" = "nononononononono"; then
+if test "$build_bitcoin_wallet$build_bitcoin_cli$build_bitcoin_tx$build_bitcoin_util$build_bitcoind$bitcoin_enable_qt$use_tests$use_bench$enable_fuzz_binary" = "nonononononononono"; then
```

Comment:
can't be merged now - maybe missing dependencies here

```
$ grep -r bitcoin_util configure.ac 
  [build_bitcoin_utils=$withval],
  [build_bitcoin_utils=yes])
  [build_bitcoin_cli=$build_bitcoin_utils])
  [build_bitcoin_tx=$build_bitcoin_utils])
  [build_bitcoin_wallet=$build_bitcoin_utils])
  build_bitcoin_utils=no
```

no build_bitcoin_util presented currently